### PR TITLE
fixed bin selected state on initial call

### PR DESCRIPTION
**Pull request by uyanga-gb**
When creating a combo chart for the first time, changing Dimensions' binning before selecting Measures is not making the selected bin Highlighted. 
Related to https://github.com/mapd/mapd-immerse/pull/4826.

### CODE REVIEW

File: dist/mapd3.css [hunk 1]
```diff
@@ -96,15 +90,9 @@
     border-collapse: separate;
     border-spacing: 0 5px; }
   .mapd3 .tooltip-group .tooltip-item, .mapd3 .legend-group .tooltip-item {
-    display: -webkit-box;
```
Comment (clhenrick) [inline dist/mapd3.css]: @uyanga-gb looks like you may have forgotten to install autoprefixer which is why these were removed. Try doing a `yarn install` and then rebuilding the `dist`

Comment (clhenrick) [reply dist/mapd3.css]: @uyanga-gb actually this is happening on a PR I have open on mapd3 as well, looking into why...

Comment (biovisualize) [reply dist/mapd3.css]: @clhenrick has a solution here (for autoprefixer to also run on dev build) https://github.com/mapd/mapd3/pull/56

Comment (clhenrick) [reply dist/mapd3.css]: yup! Just merged it into `master` so when you merge with master just make sure to rebuild `dist`

**Comment by clhenrick** [comment]
@uyanga-gb looks like you need to fix a conflict in the compiled css

**Pull request merged**